### PR TITLE
Issue #4437: Arranged the remaining files from 'imports' directory and modified the test and input files based on the rearrangement

### PR DESCRIPTION
**Pull request by subkrish**
Issue #4437 

This PR arranges the remaining files in the 'imports' directory because they are used in different files. The rearrangement also caused a few changes to made to certain input and test files.

### CODE REVIEW

File: src/test/java/com/puppycrawl/tools/checkstyle/CheckerTest.java [hunk 1]
```diff
@@ -680,7 +680,8 @@ public void testCacheAndCheckWhichAddsNewResourceLocationButKeepsSameCheckerInst
 
         final DynamicalResourceHolderCheck check = new DynamicalResourceHolderCheck();
         final String firstExternalResourceLocation = getPath("checks" + File.separator
-            + "imports" + File.separator + "import-control_one.xml");
+            + "imports" + File.separator + "importcontrol"
+            + File.separator + "InputImportControlOne.xml");
```
Comment (romani) [inline src/test/java/com/puppycrawl/tools/checkstyle/CheckerTest.java]: please copy that ml file to folder for Checker input files.
Each module should have its own input files, nothing is sharable.

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/imports/unusedimports/InputUnusedImports.java [hunk 2]
```diff
@@ -5,7 +5,7 @@
 package com.puppycrawl.tools.checkstyle.checks.imports.unusedimports;
 
 import com.puppycrawl.tools.checkstyle.checks.imports.*;
-                                     import com.puppycrawl.tools.checkstyle.checks.imports.InputImportBug;
+import com.puppycrawl.tools.checkstyle.checks.imports.unusedimports.InputUnusedImportsBug;
```
Comment (romani) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/imports/unusedimports/InputUnusedImports.java]: please restore weird indentation, this file is for testing, the more weird file the better. Just change package and name of class.

File: src/test/java/com/puppycrawl/tools/checkstyle/CheckerTest.java (hunk 1 above)
Comment (subkrish) [inline src/test/java/com/puppycrawl/tools/checkstyle/CheckerTest.java]: There is no folder yet specifically for CheckerTest. Do just copy the input xml file outside in the 'checks' directory.

Comment (romani) [inline src/test/java/com/puppycrawl/tools/checkstyle/CheckerTest.java]: Checker test is located at:
src/test/java/com/puppycrawl/tools/checkstyle/CheckerTest.java
so its inputs should be (please create folder):
src/test/resources/com/puppycrawl/tools/checkstyle/checker

Comment (subkrish) [inline src/test/java/com/puppycrawl/tools/checkstyle/CheckerTest.java]: @romani Right. But should I create it in the scope of this issue or open another PR for creating the folder for Checker? For now I was planning to just move it out to src/test/resources/com/puppycrawl/tools/checkstyle and later make a folder and move it to src/test/resources/com/puppycrawl/tools/checkstyle/checker. What do you suggest?

Comment (romani) [inline src/test/java/com/puppycrawl/tools/checkstyle/CheckerTest.java]: in scope of this issue please.
ok, copy/move it to src/test/resources/com/puppycrawl/tools/checkstyle .

Comment (subkrish) [inline src/test/java/com/puppycrawl/tools/checkstyle/CheckerTest.java]: Done.

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/imports/unusedimports/InputUnusedImports.java (hunk 2 above)
Comment (subkrish) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/imports/unusedimports/InputUnusedImports.java]: Done.

**Comment by subkrish** [comment]
@romani Why is wercker failing?

**Comment by rnveach** [comment]
@subkrish Wrecker failed on `NoExceptiontest - Checkstyle ,sevntu-checkstyle`.

````
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-site-plugin:3.3:site (default-site) on project sample: Error during page generation: Error rendering Maven report: Failed during checkstyle configuration: Exception was thrown while processing /pipeline/source/contribution/checkstyle-tester/src/main/java/checkstyle/src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/InputCustomAnnotation.java: NullPointerException -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.apache.maven.plugins:maven-site-plugin:3.3:site (default-site) on project sample: Error during page generation

Caused by: java.lang.NullPointerException
	at com.puppycrawl.tools.checkstyle.checks.sizes.MethodCountCheck$MethodCounter.access$000(MethodCountCheck.java:218)
	at com.puppycrawl.tools.checkstyle.checks.sizes.MethodCountCheck.raiseCounter(MethodCountCheck.java:140)
	at com.puppycrawl.tools.checkstyle.checks.sizes.MethodCountCheck.visitToken(MethodCountCheck.java:112)
````

**Comment by subkrish** [comment]
@rnveach I don't understand why this is happening. How can I fix this?

**Comment by rnveach** [comment]
@subkrish @romani This is bug from recent merge of indentation check improvement in https://github.com/checkstyle/checkstyle/pull/4217.
`InputCustomAnnotation` was created in that PR. I can confirm the issue in latest master outside this PR.

Issue stems from `actualCounter` in `MethodCountCheck` is null at https://github.com/checkstyle/checkstyle/blob/4a087c020c2b97130e94f9292b94cb4bc1f6ed93/src/main/java/com/puppycrawl/tools/checkstyle/checks/sizes/MethodCountCheck.java#L140.

**Comment by rnveach** [comment]
@subkrish for your work, you can ignore this error.

**Comment by subkrish** [comment]
Ok thanks.

**Pull request merged**